inference-engine/src/low_precision_transformations/src/strided_slice.cpp

```diff
@@ -45,13 +46,24 @@ std::shared_ptr<Node> stridedSliceDeqConstant(
     }
 
     const auto stridedSlice = as_type_ptr<ngraph::opset1::StridedSlice>(strSlice);
+
+    auto beginMask = stridedSlice->get_begin_mask();
+    auto endMask = stridedSlice->get_end_mask();
+    for (size_t i = 0; i < constantShape.size(); ++i) {
+        // don't slice constant if current dimension is 1
+        if (constantShape[i] == 1ul) {
+            beginMask[i] = 1ul;
+            endMask[i] = 1ul;
+        }
+    }
```

Comment:
Cover by test case, please

Reply:
This test case has been added already (Please, see `inference-engine/tests/functional/inference_engine/lp_transformations/strided_slice_transformation.cpp`)